runestone/activecode/js/activecode.js

```diff
@@ -649,7 +648,7 @@ export class ActiveCode extends RunestoneBase {
             $(this.timestampP).text(`${curVal} - ${pos + 1} of ${outOf}`);
             this.logBookEvent({
                 event: "activecode",
-                act: "slide:" + curVal,
```

Comment:
I believe this is correctly logging that the history slider is changing positions.  Why are you changing the act to save&Run ??

Reply:
Turned it back to "slide"